release-schema.json

```
@@ -14,7 +14,45 @@
         "legalBasis": {
           "title": "Legal basis",
           "description": "The legal basis of the procedure.",
-          "$ref": "#/definitions/Classification"
+          "type": "object",
+          "properties": {
+            "scheme": {
+              "title": "Scheme",
+              "description": "The scheme or codelist from which the legislation's code is taken.",
+              "type": [
+                "string",
+                "null"
+              ],
+              "codelist": "+itemClassificationScheme.csv",
+              "openCodelist": true,
+              "minLength": 1
+            },
+            "id": {
+              "title": "ID",
+              "description": "The legislation's code taken from the scheme.",
+              "type": [
+                "string",
+                "integer",
+                "null"
+              ],
```

Now that we're no longer reusing the `Classification` definition, we can remove the erroneous `integer` type.

```suggestion
              "type": [
                "string",
                "null"
              ],
```